### DISCUSSION TITLE
Is there an up-to-date Advanced Intro that covers v3.2.x?

### DISCUSSION THREAD

**Discussion by wrogner** [explosion/spaCy#9822] [Help: Other Questions] [answered]
Chapter 4 of Advanced NLP with Space covers the training loop.

While in previous chapters, differences between Space 2.x (version of the intro) and 3.x could be resolved by reading the documentation, chapter 4 from step 5 onwards deals with training new models.

The problems there cannot be fixed by simply looking up parameters. There are some new concepts that are hidden from the novice.

The examples cannot be corrected with pre Python and up to this step Space knowledge.

Is there an update somewhere or a tutorial where one can implement a training loop?

## Which page or section is this issue related to?

relates to:

https://course.spacy.io/en/chapter4

**Comment by adrianeboyd** [comment]
Hi, we're working on an updated version of the course for v3. I think we're still waiting on translations for a few languages, but look for an announcement soon!

**Comment by wrogner** [comment, marked as answer]
Not very helpful as I need this for my lectures (I would not want to go back to nltk).

Is there a brief intro to training models (that would work)?

**Reply by polm** [reply]
The [training section of the docs](https://spacy.io/usage/training#basics) covers how to train models. If you want a more concrete example you can look at the [example projects](https://github.com/explosion/projects/tree/v3/tutorials).

**Comment by wrogner** [comment]
Thank you for the link.

The very last entry on this page solved my issue ;-)

Let's see, if I can finish my task with this.

Thanks, Wolf

**Comment by polm** [comment]
Note that the course has been updated to cover v3 all through at this point. 

https://course.spacy.io/
